.env.oss

```diff
@@ -14,3 +14,5 @@ METAPHYSICS_ENDPOINT=https://metaphysics-staging.artsy.net
 USER_ID=
 USER_ACCESS_TOKEN=
 
+# `cd path/to/force && hokusai staging env get | grep -i stripe`
+STRIPE_PUBISHABLE_KEY=
```

Comment:
Perhaps move this up above the user specific vars to the rest of the env vars and make the comment generic so it applies to all those vars?